src/pdsh/opt.c

```diff
@@ -66,7 +66,8 @@
 
 #define OPT_USAGE_DSH "\
 Usage: pdsh [-options] command ...\n\
--S                return largest of remote command return values\n"
+-S                return largest of remote command return values\n\
+-k                fail fast on ssh failures or non-zero return code\n"
```

Comment:
```suggestion
-k                fail fast on connect failures or non-zero return code\n"
```

Pdsh doesn't only support `ssh` so let's keep the option description generic.